packages/web/src/config.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Instrumentation,
+  MetaItem,
+  App,
+  Session,
+  User,
+  Config,
+  defaultGlobalObjectKey,
+  Transport,
+} from '@grafana/agent-core';
+
+import { ErrorsInstrumentation, WebVitalsInstrumentation } from './instrumentations';
+import { browserMeta, pageMeta } from './metas';
+import { FetchTransport } from './transports';
+
+export interface BrowserConfig {
+  url?: string;
+  apiKey?: string;
+  app: App;
+  session?: Session;
+  user?: User;
+  globalObjectKey?: string;
+  preventGlobalExposure?: boolean;
+  metas?: MetaItem[];
+  instrumentations?: Instrumentation[];
+  transports?: Transport[];
+}
+
+export const defaultMetas: MetaItem[] = [browserMeta, pageMeta];
+
+export const getDefaultInstrumentations = (): Instrumentation[] => [
+  new ErrorsInstrumentation(),
+  new WebVitalsInstrumentation(),
+];
+
+export function makeCoreConfig(browserConfig: BrowserConfig): Config {
+  const transports: Transport[] = [];
+  if (browserConfig.transports) {
```

Comment:
Why not both here? Use case: as a developer i might have the fetch transport set via the `browserConfig.url` property but I might temporarily need to enable another transport. Because we limit the availability of `browserConfig.url` with ``browserConfig.transports`, I will manually need to install the fetch transport package and add it to the transports list.

Reply:
it's all in the `agent-web` package, no separate package. My goal here was to avoid confusion and be explicit about what config will be actual